src/Microsoft.ML.Data/MLContext.cs

```diff
@@ -10,8 +10,9 @@
 namespace Microsoft.ML
 {
     /// <summary>
-    /// The <see cref="MLContext"/> is a starting point for all ML.NET operations. It is instantiated by user,
-    /// provides mechanisms for logging and entry points for training, prediction, model operations etc.
+    /// The starting point for all ML.NET applications. Once instantiated by the user, it
```

Comment:
ML.NET Operations is more correct.  (Applicatoin is a hosting app that contains ML.NET components.)
"The common context for all ML.NET operations.  Once instantiated by the user it provides a way to create variety of components for data preparation.... etc.  It also allows logging, execution control and ability set repeatable random numbers